### ISSUE TITLE
Add `--format` option to `multipass version` command

### ISSUE THREAD

**Issue by cjp256** [canonical/multipass#2020]
**Describe the bug**
Multipass sometimes logs unrelated messages to stdout.  When parsing stdout information, this can lead to undesirable behavior.

**To Reproduce**
Install multipass 1.5 for Windows.

Run `multipass version`

How, and what happened?

```
>multipass version 2>stderr.txt
multipass  1.5.0+win
multipassd 1.5.0+win

###########################################################################################
New Multipass 1.6.2 release
Now with bridged networking support

Go here for more information: https://github.com/CanonicalLtd/multipass/releases/tag/v1.6.2
###########################################################################################
```
**Expected behavior**
The extra outputs should be logged to stderr to allow for more reliable parsing of stdout.

**Comment by Saviq** [comment]
I'd say we should have a `--format` for `multipass version` instead, so you can safely parse that.

**Comment by cjp256** [comment]
> 
> 
> I'd say we should have a `--format` for `multipass version` instead, so you can safely parse that.

Even better :+1:

**Comment by surahman** [comment]
**_With respect to #2077 :_**
I made the modifications to include the `--format` flag and it will only be recognised in conjunction with the `-V` and `--version` cmdl flags. Corrected a typo with a missing quote in the help info string.

**Comment by Saviq** [comment]
Fixed in #2077
